### PR TITLE
dracula-theme: 1.3.0 -> 2.0 and rename ant-dracula to dracula-theme

### LINKED ISSUE #97655: Rename ant-dracula-theme → gtk-dracula-theme
[Ant-Dracula] has moved to [Dracula].

[Dracula]: https://draculatheme.com
[Ant-Dracula]: https://github.com/EliverLara/Ant-Dracula

Homepage: https://draculatheme.com/gtk
Repository: https://github.com/dracula/gtk

### PR DESCRIPTION
**Pull request by Vonfry**
###### Motivation for this change
Update package and rename it due to the upstream repo has been renamed.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc: @alexarice 
fixes: #97655

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -9088,6 +9088,12 @@
     githubId = 508305;
     name = "Jaroslavas Pocepko";
   };
+  vonfry = {
```
Comment (alexarice) [inline maintainers/maintainer-list.nix:9091]: Can the change to `maintainer-list.nix` be in a separate commit titled "maintainers: add vonfry"

File: pkgs/data/themes/dracula-theme/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  src = fetchFromGitHub {
+    owner = "dracula";
+    repo = "gtk";
+    rev = "v${version}";
+    sha256 = "10j706gnhdplhykdisp64vzzxpzgn48b5f1fkndcp340x7hf2mf3";
+  };
+
+  propagatedUserEnvPkgs = [
+    # gtk-engine-murrine
```
Comment (alexarice) [inline pkgs/data/themes/dracula-theme/default.nix]: If this isn't needed this shouldn't be here

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -18767,7 +18767,7 @@ in
 
   ant-bloody-theme = callPackage ../data/themes/ant-theme/ant-bloody.nix { };
 
-  ant-dracula-theme = callPackage ../data/themes/ant-theme/ant-dracula.nix { };
+  dracula-theme = callPackage ../data/themes/dracula-theme { };
```
Comment (alexarice) [inline pkgs/top-level/all-packages.nix:18770]: This will break existing configurations. Maybe there should be an error thrown when using the old package that tells you to use the new one and to change the theme name in your config

**Review by alexarice — CHANGES_REQUESTED** [review]
nix-review passes. Have left some comments

File: pkgs/data/themes/dracula-theme/default.nix (hunk 2 above)
Comment (alexarice) [reply pkgs/data/themes/dracula-theme/default.nix]: Think you've already removed this

File: pkgs/top-level/aliases.nix [hunk 4]
```diff
@@ -738,4 +738,7 @@ mapAliases ({
   /* Cleanup before 21.03 */
   riot-desktop = throw "riot-desktop is now element-desktop!";
   riot-web = throw "riot-web is now element-web";
+
+  ant-dracula-theme = throw "ant-dracula-theme is now dracula-theme";
```
Comment (alexarice) [inline pkgs/top-level/aliases.nix:742]: Can this mention that the theme name has changed. For example I will have to change my config [here](https://github.com/alexarice/dotfiles/blob/f10a92b45730adf87b1f39e7c891ab25cfe15fc8/dotfiles/gtk.nix#L13)

**Review by alexarice — CHANGES_REQUESTED** [review]
Is this the sort of change that should have a release note?

**Review by alexarice — APPROVED** [review]
Looks good to me, Thanks for doing this fix, I wasn't even aware there had been an update.
I'll look into release notes, but other than that nix-review passes and looks good to go

File: pkgs/data/themes/dracula-theme/default.nix [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  themeName = "Dracula";
+  version = "2.0";
+in
+stdenv.mkDerivation rec {
```
Comment (alexarice) [inline pkgs/data/themes/dracula-theme/default.nix:7]:
```suggestion
stdenv.mkDerivation {
```
Sorry, small change I missed

**Review by alexarice — APPROVED** [review]
Asked on irc and I don't think there's any need for release notes so should be good as is

File: pkgs/data/themes/ant-theme/ant-dracula.nix [hunk 6]
```diff
@@ -1,34 +0,0 @@
...
-  version = "1.3.0";
-
-  src = fetchurl {
-    url = "https://github.com/EliverLara/${themeName}/releases/download/v${version}/${themeName}.tar";
-    sha256 = "00b8w69xapqy8kc7zqwlfz1xpld6hibbh35djvhcnd905gzzymkd";
-  };
-
-  propagatedUserEnvPkgs = [
-    gtk-engine-murrine
-  ];
```
Comment (romildo) [inline pkgs/data/themes/ant-theme/ant-dracula.nix:17]: Why is `gtk-engine-murrine` removed? It seems that the gtk2 theme still depends on this engine.

Comment (Vonfry) [reply pkgs/data/themes/ant-theme/ant-dracula.nix:17]: Sorry, I checked the code, it uses this in gnome-terminal, which I don't use. I readd it.

File: pkgs/top-level/all-packages.nix (hunk 3 above)
Comment (alexarice) [reply pkgs/top-level/all-packages.nix:18770]: I believe the alias should add an appropriate warning
